src/commands/kv/mod.rs

```diff
@@ -63,8 +63,8 @@ pub fn get_namespace_id(target: &Target, binding: &str) -> Result<String, failur
     )
 }
 
-fn api_client(user: GlobalUser) -> Result<HttpApiClient, failure::Error> {
-    Ok(HttpApiClient::new(Credentials::from(user)))
+fn api_client(user: &GlobalUser) -> HttpApiClient {
+    HttpApiClient::new(Credentials::from(user.to_owned()))
```

Comment:
I don't think we need to return a Result here; we don't seem to unwrap anything here.

Reply:
hmmm. Legit!